recipe/meta.yaml

```diff
@@ -27,54 +33,55 @@ requirements:
     - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - scons >=4.1
-    - setuptools
+    - packaging
   host:
     - python >=3.8
-    - libboost-devel
     - numpy
+    - libboost-devel
     - fmt
     - eigen
     - yaml-cpp
-    - sundials >=6.7.0
+    - sundials ==7.0.0
```

Comment:
Can we build against the latest release of sundials here?

Reply:
I don't know how this is supposed to be done. In a perfect world, I assume we would build against 7.0.0 and that would be compatible with 7.x.y at runtime, but that hasn't necessarily worked in the past.

Reply:
Sorry, I should have been more clear. My suggestion was to change this to `==7.2.0`

Reply:
Do we need to set a version here other than `<8`? Once this builds with 7.2.0, I think the `run_exports` setting in the sundials feedstock will make the cantera package depend on Sundials 7.2.x. Optimistically, then, all it would take to support future 7.y releases would be a rebuild with an incremented build number.

Reply:
I'm not sure why this was set to `==`. Maybe it comes from a pre-conda-build-3 state of the recipe before run exports were added. It may also indicate the runtime limitations more clearly to anyone who looks at the recipe. If one didn't know that sundials has run exports, it'd be pretty confusing why you couldn't install sundials 7.3 with this version of Cantera. 

Anyhow, a change to <8 seems fine to me, thank you!